Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt

```diff
@@ -0,0 +1,287 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Staged Rollout users/groups and sign-ins\r\n"
+      },
+      "name": "text - 10"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AuditLogs \r\n| where OperationName contains 'feature'\r\n        and OperationName contains 'user'\r\n        and OperationName contains 'add'\r\n| where TimeGenerated > ago(30d) \r\n| make-series count(OperationName) \r\n    on TimeGenerated in range(ago(30d), now(), 1d) by OperationName\r\n\r\n\r\n\r\n",

```

Comment:
Consider passing in time as a parameter, please take a look at the other workbooks for examples